superset-frontend/src/SqlLab/components/SqlEditor/index.tsx

```diff
@@ -880,15 +880,17 @@ const SqlEditor: FC<Props> = ({
               startQuery={startQuery}
             />
           )}
-          <AceEditorWrapper
-            autocomplete={autocompleteEnabled}
-            onBlur={onSqlChanged}
-            onChange={onSqlChanged}
-            queryEditorId={queryEditor.id}
-            onCursorPositionChange={handleCursorPositionChange}
-            height={`${aceEditorHeight}px`}
-            hotkeys={hotkeys}
-          />
+          {isActive && (
```

Comment:
@justinpark Isn't possible to just make the ace editor stop listening to events when inactive instead of removing it from the DOM? That way you would retain the previous performance gains.

Reply:
@michael-s-molina That approach involves many aspects that need to be checked, such as regression, making it challenging to implement as a hotfix.
It would be better to revisit and resolve the issue related to the performance of the Ace Editor (if we feel downgrade) after addressing the current problem.